packages/web-components/fast-components/src/file/index.ts

```diff
@@ -0,0 +1,56 @@
+import { html, repeat } from "@microsoft/fast-element";
+import { File, fileTemplate as template } from "@microsoft/fast-foundation";
+import { FileOptions } from "@microsoft/fast-foundation";
+import { fileStyles as styles } from "./file.styles";
+
+/**
+ * The default FAST File Template for a file-list
+ *  @public
+ */
+export const defaultFileListTemplate = html<File>`
```

Comment:
It feels like the default templates should go in foundation so the component is usable in its default configuration. As it is now it looks like there would be no button and no place to list files, and thus just two slots always requiring external configuration. We could template in a button and provide some way to set the caption which would support most basic uses of this control type. Likewise the generic `ul` / `li` structure should support most if not all needs.

Reply:
Makes sense. And yes, `ul`/`li` as a default is what will ship with `fast-file`.

Reply:
This is a great thought @bheston, that would make composition easy out of the box while maintaining the slotted content customization. Love it.